packages/contracts-bsc/contracts/common/BEP20.sol

```diff
@@ -1,30 +1,13 @@
-/**
- * @notice This is a copy of openzeppelin ERC20 contract with removed state variables.
- * Removing state variables has been necessary due to proxy pattern usage.
```

Comment:
Let's at least keep the part of this comment that talks about moving storage variables to `ProxyStorage`. (Feel free to change ERC20 to BEP20, mutatis mutandis). Otherwise users may be unaware of the design pattern used for TrueUSD. If they mistakenly try adding storage variables outside of `ProxyStorage` in a future refactor, then they may cause storage corruption.

I would also add that BEP's `getOwner()` points to `owner`, since that is a new change introduced by this commit.